common/app/conf/switches.scala

```diff
@@ -77,299 +82,550 @@ object Switches {
+  val ExternalVideoEmbeds = Switch(
+    "Performance",
+    "external-video-embeds",
     "If switched on then we will accept and display external video views",
-    safeState = Off, sellByDate = never
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
   )
 
-  val DiscussionSwitch = Switch("Performance", "discussion",
+  val DiscussionSwitch = Switch(
+    "Performance",
+    "discussion",
     "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
-    safeState = On, sellByDate = never
+    safeState = On,
+    sellByDate = never,
+    exposeClientSide = true
   )
 
-  val DiscussionPageSizeSwitch = Switch("Performance", "discussion-page-size",
+  val DiscussionPageSizeSwitch = Switch(
+    "Performance",
+    "discussion-page-size",
     "If this is switched on then users will have the option to change their discussion page size",
-    safeState = Off, sellByDate = never
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
   )
 
-  val OpenCtaSwitch = Switch("Performance", "open-cta",
-    "If this switch is on, will see a CTA to comments on the right hand side. Turn this off if the Open API is blowing up.",
-    safeState = Off, sellByDate = never
+  val OpenCtaSwitch = Switch(
+    "Performance",
+    "open-cta",
+    "If this switch is on, will see a CTA to comments on the right hand side. Turn this off if the Open API " +
+      "is blowing up.",
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
   )
 
-  val ImageServerSwitch = Switch("Performance", "image-server",
+  val ImageServerSwitch = Switch(
+    "Performance",
+    "image-server",
     "If this switch is on images will be served off i.guim.co.uk (dynamic image host).",
-    safeState = On, sellByDate = never
+    safeState = On,
+    sellByDate = never,
+    exposeClientSide = false
   )
 
-  val PngResizingSwitch = Switch("Performance", "png-resizing",
+  val PngResizingSwitch = Switch(
+    "Performance",
+    "png-resizing",
     "If this switch is on png images will be resized via the png-resizing server",
-    safeState = Off, sellByDate = never
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = false
   )
-
-  // Commercial
-  val DfpCachingSwitch = Switch("Commercial", "dfp-caching",
+  
+  val DfpCachingSwitch = Switch(
+    "Commercial",
+    "dfp-caching",
     "Have Admin will poll DFP to precache adserving data.",
-    safeState = On, sellByDate = never
+    safeState = On,
+    sellByDate = never,
+    exposeClientSide = false
   )
 
-  val CommercialSwitch = Switch("Commercial", "commercial",
+  val CommercialSwitch = Switch(
+    "Commercial",
+    "commercial",
     "If this switch is OFF, no calls will be made to the ad server. BEWARE!",
-    safeState = On, sellByDate = never
+    safeState = On,
+    sellByDate = never,
+    exposeClientSide = false
```

Comment:
This one is checked in javascriptLaterSteps to initialise commercial scripts so it needs to be exposed to the client side